src/cloudRun.ts

```diff
@@ -299,4 +303,58 @@ export class CloudRun {
       this.methodOptions,
     );
   }
+
+  async pollService(serviceResponse: run_v1.Schema$Service): Promise<string> {
+    let url = getUrl(serviceResponse);
+    while (!getReadyStatus(serviceResponse) && !url) {
```

Comment:
Shouldn't we have a timeout incase it is never ready?

Reply:
done